vwtags.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python3
```

Comment:
Why `/usr/bin/python3` isn't `/usr/bin/env python3` better for compatibility with different distributions?

Reply:
That's actually fine with me, updated.